models/stocks.js

```diff
@@ -1,5 +1,7 @@
 const firestore = require('../utils/firestore')
 const stocksModel = firestore.collection('stocks')
+const userStocksModel = firestore.collection('user-stocks')
```

Comment:
We're using `stocks` right? The firestore collection

Reply:
`stocks` collection only contains the details of the stocks which the users can see on screen and trade them.
`user-stocks` collection contains the details of the stocks the individual users have

The API contracts need to be updated for this, will be doing that soon.